### PR TITLE
feat: added routes layout

### PR DESCRIPTION
**Pull request by RomanRiyanov**
Added layout and routes of base components. Of course, it should be removed as soon as possible.

### CODE REVIEW

**Review by dev0x451 — CHANGES_REQUESTED** [review]

File: src/index.css [hunk 1]
```diff
@@ -1,71 +1,16 @@
...
-  }
-  a:hover {
-    color: #747bff;
-  }
-  button {
-    background-color: #f9f9f9;
-  }
+  width: 100%;
+  height: 100%;
+  --nav-color: #276CB1;
```
Comment (dev0x451) [inline src/index.css:15]: css переменные чуток не так работают, к примеру:
:root {
  /* Colors */
  --color-primary-100: #f4ebff;
}

далее где-то:

backbround-color: var(--color-primary-100)

File: src/main.tsx
[the remark this answers is not in the document]
Comment (RomanRiyanov) [reply src/main.tsx:1]: Need to be done

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+      </Routes>
+
+      {/* <Navbar/> */}
+      {/* <Homepage/> */}
+    </main>
+  )
+}
+
+export default App
```
[the remark this answers is not in the document]
Comment (RomanRiyanov) [reply src/components/App/App.tsx]: Done

File: src/index.css (hunk 1 above)
Comment (RomanRiyanov) [reply src/index.css]: Done

**Review by dev0x451 — APPROVED** [review]

**Pull request merged**